.eslintignore

```diff
@@ -0,0 +1 @@
+flow-typed/npm
```

Comment:
This was painful: they're not proper JS files so:

1.    Eslint was outputting errors at commit time (like "Type annotations require valid Flow declaration flowtype/no-types-missing-file-annotation")
 2.   prettier kept rewriting them even when I wanted to make only a small change

An alternative could be a specific .eslintrc for this directory but for now I don't think this is useful.

Reply:
I'm happy with ignoring it for now.